app/views/customers/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1><%= @customer.name %></h1>
+
+<h2> Items </h2>
+<ul>
+  <% @customer.items.each do |item| %>
+    <li>
+      Supermarket: <%= item.supermarket.name %>
+      Item: <%= item.name %>
+      Price: <%= item.price %>
```

Comment:
Nice use of labels. You can also be using these labels in your test to make sure the Item name is under the Item label and the item.price is next to the Price label.